mysql_init.sls

```diff
@@ -0,0 +1,95 @@
+install_mysql:
+  pkg.installed:
+    - pkgs:
+      - mysql-server
+      - python-mysqldb 
+
+reload-mysql:
+  service.running:
+    - name: mysql
+    - enable: True
+    - reload: True
+
+root_user:
+  mysql_user.present:
+    - name: 'root'
+    - password: 'TOPSECRET'
+
+mysql remove anonymous users:
+  mysql_user.absent:
+    - name: ''
+    - host: 'localhost'
+    - connection_user: 'root'
+    - connection_pass: 'TOPSECRET'
+    - connection_charset: utf8
+
+mysql remove test database:
+  mysql_database.absent:
+    - name: test
+    - host: 'localhost'
+    - connection_user: 'root'
+    - connection_pass: 'TOPSECRET'
+    - connection_charset: utf8
+
+create api database:
+  mysql_database.present:
+    - name: api_production
```

Comment:
maybe obs_api_production?

Reply:
You could also move that value into a variable to avoid duplication

Reply:
Thanks, I'll try to save it into a pillar as I need that in obs-server.sls also.